features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesStateProvider.kt

```diff
@@ -99,7 +99,7 @@ fun aMessagesState(
     userHasPermissionToRedactOther: Boolean = false,
     userHasPermissionToSendReaction: Boolean = true,
     composerState: MessageComposerState = aMessageComposerState(
-        richTextEditorState = RichTextEditorState("Hello", initialFocus = true),
+        richTextEditorState = RichTextEditorState(initialHtml = "Hello", initialMarkdown = "Hello", initialFocus = true),
```

Comment:
I suggest that you create a method `aRichTextEditorState(initialText: String)` that would avoid coping the parameter value everywhere.

Reply:
Yeah, good call.